.github/workflows/platform.librarycheck.psrule.yml

```diff
@@ -0,0 +1,108 @@
+name: '.Platform: Library PSRule pre-flight validation'
```

Comment:
Question: Should we have an ADO counterpart?

Reply:
We should eventually, but we're currently blocked on that for the same reasons described by issue #2427